charts/dependabot-gitlab/README.md

```diff
@@ -119,6 +119,13 @@ By default chart installs instance of [redis](https://github.com/bitnami/charts/
 | worker.probePort | int | `7433` |  |
 | worker.replicaCount | int | `1` | Worker container replicas count |
 | worker.resources | object | `{}` | Worker container resource definitions |
+| worker.serviceMonitor.additionalLabels | object | `{}` |  |
```

Comment:
Definition of `serviceMonitor` should not be under `worker` only, since both web and worker expose the `/metrics` endpoint.

It would be better to move it under `metrics` key I added recently.